### PR TITLE
Add more attraction presets

### PR DESCRIPTION
**Pull request by hikemaniac**
This PR adds presets for

- attraction=kiddie_ride
- attraction=log_flume
- attraction=swing_carousel

according to the [wiki](https://wiki.openstreetmap.org/wiki/Key:attraction).

### CODE REVIEW

File: data/presets/presets/attraction/kiddie_ride.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "payment_multi_fee",
+        "charge_fee"
+    ],
+    "moreFields": [
+        "{attraction}",
+        "max_age",
+        "min_age"
+    ],
+    "geometry": [
+        "point"
```
Comment (bhousel) [inline data/presets/presets/attraction/kiddie_ride.json]: Should this one also have "area" ?

File: data/presets/presets/attraction/log_flume.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-tower",
```
Comment (bhousel) [inline data/presets/presets/attraction/log_flume.json]: We should probably pick a better icon for a log flume.. Maybe `maki-ferry` for now until we come up with a better one.

File: data/presets/presets/attraction/kiddie_ride.json (hunk 1 above)
Comment (hikemaniac) [reply data/presets/presets/attraction/kiddie_ride.json]: I'd agree with the wiki here. Usually, those kiddy rides are just small horses, cars or helicopters moving to and fro. In my opinion they are too small to be mapped as areas.

Comment (bhousel) [reply data/presets/presets/attraction/kiddie_ride.json]: Oh ok!  Didn't realize that's what it was for.

File: data/presets/presets/attraction/log_flume.json (hunk 2 above)
Comment (hikemaniac) [reply data/presets/presets/attraction/log_flume.json]: Thank you for pointing that out, I just pushed a commit changing the icon. 👍

**Pull request merged**